frontend/src/features/dataset/DatasetPage.tsx

```diff
@@ -36,15 +36,35 @@ const DatasetPage: React.FC<{}> = ({
     dispatch(loadDataset(ref))
   }, [dispatch, qriRef.username, qriRef.name, qriRef.path, isNew])
 
+  const { pathname } = useLocation();
+
+  const isPreview = pathname.includes('preview')
+
   return (
     <div className='flex flex-col h-full w-full' style={{ backgroundColor: '#f3f4f6'}}>
       <NavBar />
       <div className='flex overflow-hidden w-full'>
         <DatasetNavSidebar qriRef={qriRef} />
-        <div className='flex flex-col flex-grow overflow-hidden'>
-          <DatasetHeader qriRef={qriRef} editable={editable} />
-          {children}
-        </div>
+        {
+          // different container scroll behavior and header border for dataset preview
+          (isPreview) && (
+            <div className='overflow-y-scroll overflow-x-hidden flex-grow relative flex'>
+              <div className='max-w-screen-lg mx-auto py-9 flex-grow'>
+                <DatasetHeader qriRef={qriRef} editable={editable} noBorder />
+                {children}
+              </div>
+            </div>
+          )
+        }
+        {
+          (!isPreview) && (
+            <div className='flex flex-col flex-grow overflow-hidden px-7 py-9'>
+              <DatasetHeader qriRef={qriRef} editable={editable} />
+              {children}
+            </div>
+          )
+        }
```

Comment:
The changes to this component are erroneously introducing a "shadow router" into the `DatasetPage` component. Instead:

* leave `DatasetPage` unchanged
* drop the `DatasetPage` wrapper in the preview route within `DatasetRoutes`
* rename `DatasetPreview` to `DatasetPreviewPage` & pull any logic & page wrapping from this file into the `DatasetPreviewPage` component

This'll also help when we wire up the API, which hits the _preview_ endpoint, instead of the full dataset endpoint.